### ISSUE TITLE
Evilcraft and Bonsai Trees

### ISSUE THREAD

**Issue by UnrefinedCatalyst** [thraaawn/BonsaiTrees#127]
I'm currently attempting to make Evilcraft compatible with bonsai trees, I've written the required JSON files for undead trees to be added to bonsai, and for blood waxed coal to be useable within bonsai pots as a soil type.

I figured that being able to grow undead trees in a regular dirt filled bonsai pot was a little overpowered, so I made it so that they required blood waxed coal as a soil, and so that the undead trees took a little longer to grow than regular bonsai trees. Although, I also made it so that blood waxed coal grows trees 25% faster, and makes them produce 25% more items.

The only issue that I have, is that in the very first section of the file for the undead tree itself, I do not have a proper name. The current placeholder I've been using is evilcraft:undeadtree, but I know there's a model name, or maybe some json file for the tree schematic itself, and currently there is no model within the bonsai pot. Functionally, it's fine, but it is hard to tell whether or not I have anything in the pot in the first place until I start getting drops from the bonsai tree.

I've attached txt files, they are fully functional when I converted them to json files in github, feel free to use them.

I've uploaded these same files to the evilcraft GitHub, and they recommended I send them to you as well.

Hope something comes of this,

Catalyst
[evilcraft_undeadtree.txt](https://github.com/thraaawn/BonsaiTrees/files/3748058/evilcraft_undeadtree.txt)
[blood_waxed_coal_block.txt](https://github.com/thraaawn/BonsaiTrees/files/3748059/blood_waxed_coal_block.txt)

**Comment by UnrefinedCatalyst** [comment]
forgot the link to the evilcraft page,
https://github.com/CyclopsMC/EvilCraft/issues/792

**Comment by Schadrach** [comment]
Just so happens I'd made configs to do undead, wisdom, both Forbidden and Arcanus trees and all the Bewitchment trees recently.  Here's the complete set, some of the models aren't ideal but I was lazy:
[bonsaiconfigs.zip](https://github.com/thraaawn/BonsaiTrees/files/3956252/bonsaiconfigs.zip)


If you just want the undead sapling models, put these in shapes.d, make sure the "name" in your types.d file matches the "type" in these files:
[undeadtree.zip](https://github.com/thraaawn/BonsaiTrees/files/3956233/undeadtree.zip)

Basically the types.d files control how the bonsais function, shapes.d how they look.  There can be multiple shapes.d files for one bonsai, if there are the bonsai will choose one at random to determine how the bonsai looks in the pot.  You can look at the format and adjust the appearance if you want, it's not hard.

**Comment by thraaawn** [comment]
I will include @Schadrach bonsaiconfigs in the next release.
